modules/admixerIdSystem.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * This module adds AdmixerId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/admixerIdSubmodule
+ * @requires module:modules/userId
+ */
+
+import * as utils from '../src/utils.js'
+import { ajax } from '../src/ajax.js';
+import { submodule } from '../src/hook.js';
+import {getStorageManager} from '../src/storageManager.js';
+
+export const storage = getStorageManager();
+
+/** @type {Submodule} */
+export const admixerIdSubmodule = {
+  /**
+   * used to link submodule with config
+   * @type {string}
+   */
+  name: 'admixerId',
+  /**
+   * used to specify vendor id
+   * @type {number}
+   */
+  gvlid: 511,
+  /**
+   * decode the stored id value for passing to bid requests
+   * @function
+   * @param {string} value
+   * @returns {{__adm__admixer:string}}
+   */
+  decode(value) {
+    return { '__adm__admixer': value }
```

Comment:
This is the value that gets exposed to bidder adapters...to follow standards it should be `admixerId`